### PR TITLE
Adding fix for parameters, adding other ivs regions

### PR DESCRIPTION
**Pull request by wizage**
*Issue #, if available:* #139 

*Description of changes:* Adding support to multiple regions.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
-  },
-  "bugs": {
-    "url": "https://github.com/awslabs/amplify-video/issues"
-  },
-  "homepage": "https://github.com/awslabs/amplify-video#readme"
-}
+{
+  "name": "amplify-category-video",
+  "version": "3.1.1",
+  "description": "Plugin for Amplify to add support for live streaming. Made for Unicorn Trivia Workshop",
```
Comment (smp) [inline package.json:4]: we should update this description to something more accurate

File: provider-utils/awscloudformation/cloudformation-templates/ivs-helpers/LambdaFunctions/IVSShim/package.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
   "name": "starfruitchannellambda",
   "version": "1.0.0",
   "description": "Create and delete Starfruit Channels",
```
Comment (smp) [inline provider-utils/awscloudformation/cloudformation-templates/ivs-helpers/LambdaFunctions/IVSShim/package.json:4]: remove all references to starfruit if possible

File: provider-utils/awscloudformation/default-values/livestream-defaults.json [hunk 3]
```diff
@@ -4,24 +4,24 @@
...
+    "securityGroup": "0.0.0.0/0",
+    "ingestType": "RTMP_PUSH",
+    "encodingProfile": "FULL",
+    "autoStart": "YES"
+  },
+  "mediaPackage": {
+    "endpoints": "HLS,DASH",
+    "startOverWindow": "86400"
+  },
+  "mediaStorage": {
```
Comment (smp) [inline provider-utils/awscloudformation/default-values/livestream-defaults.json:18]: mediaStore would be a better name, consistent with service name (i know these edits came from linting, but  still trying to call out minor things)

File: provider-utils/awscloudformation/service-walkthroughs/ivs-push.js [hunk 4]
```diff
@@ -8,47 +10,56 @@ module.exports = {
...
+      choices: questions.channelQuality.options,
+      default: defaults.channel.channelQuality,
     },
     {
-      type: inputs[2].type,
-      name: inputs[2].key,
-      message: inputs[2].question,
-      choices: inputs[2].options,
-      default: 'LOW',
+      name: 'chanelLatency',
```
Comment (smp) [inline provider-utils/awscloudformation/service-walkthroughs/ivs-push.js:52]: typo

File: provider-utils/awscloudformation/service-walkthroughs/vod-push.js [hunk 5]
```diff
@@ -195,6 +195,12 @@ async function serviceQuestions(context, options, defaultValuesFilename, resourc
         ],
       },
     };
+  } else {
+    props.parameters = {
+      authRoleName: {
```
Comment (smp) [inline provider-utils/awscloudformation/service-walkthroughs/vod-push.js:200]: intentional lowerCamelCase vs UpperCamelCase on line 201?

**Review by smp — CHANGES_REQUESTED** [review]
Some minor comments, one major typo

File: package.json (hunk 1 above)
Comment (wizage) [reply package.json:4]: Yes! We should! Lets do it

File: provider-utils/awscloudformation/service-walkthroughs/vod-push.js (hunk 5 above)
Comment (wizage) [reply provider-utils/awscloudformation/service-walkthroughs/vod-push.js:200]: Intentional. (required way by amplify)

File: provider-utils/awscloudformation/default-values/livestream-defaults.json (hunk 3 above)
Comment (wizage) [reply provider-utils/awscloudformation/default-values/livestream-defaults.json:18]: Probably best left for #1

**Comment by wizage** [comment]
Super broken

**Pull request closed without merging**